contracts/RlnBase.sol

```diff
@@ -118,6 +124,7 @@ abstract contract RlnBase {
 
         members[idCommitment] = idCommitmentIndex;
```

Comment:
can we add a `mapping(uint256 => uint256) public indexToMember;` that stores the index->idCommitment? Similar to the existing `mapping(uint256 => uint256) public members;` but the other way around. Perhaps redundant and can be optimized? I'm ok by being redundant for the PoC.

We would need also a getter for that. Something like:
```
getCommitment(uint256 index) { 
   return indexToMember[index];
}
```

So a client can easily sync the tree by:
* for i in idCommitmentIndex: getCommitment(i)
* And then just monitoring `idCommitmentIndex` and getting the memberships that are missing.

And also a getter for `idCommitmentIndex` I guess.

Reply:
addressed!